### PR TITLE
stalled on CSS

### PR DESCRIPTION
**Pull request by datboyakin**
got hard stopped on CSS. In the end, I've returned to vanilla over a library.

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        </Container>
+        <Container className="articleComments">
+          <p>comments go here</p>
+        </Container>
+        <Button
+          type="button"
+          onClick={() => {
+            handleBackToArticles();
+          }}
+          value={article.title}
```
Comment (P-Copley) [inline src/components/Article.jsx:41]: You don't need to add value's to buttons. This can be removed 👍

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                </h5>
+                <small>Author {article.author}</small>
+                <p>Category {article.topic}</p>
+                <p>Likes {article.votes}</p>
+                <Button
+                  type="button"
+                  onClick={() => {
+                    handleReadArticle(article.article_id);
+                  }}
+                  value={article.title}
```
Comment (P-Copley) [inline src/components/Articles.jsx:42]: Same as above

File: src/components/Article.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  console.log(article_id, 'article ID in Article');
+
+  useEffect(() => {
+    getArticleById(article_id).then(({ data }) => {
+      setArticle(data.article);
+    });
+  }, [article_id]);
+
+  const handleBackToArticles = () => {
+    console.log('inside button click');
```
[the remark this answers is not in the document]
Comment (datboyakin) [reply src/components/Article.jsx:20]: yes... That's actually at a deeper page than the 'articles' category page. I'll be handling the logic for that later.

**Comment by P-Copley** [comment]
Looking great so far. Just a couple of logs left lying around but the code looks good 👍 
No need to go overboard with the CSS libraries. Just having the buttons with your own CSS works well here.

**Pull request merged**